packages/@aws-cdk/aws-stepfunctions-tasks/test/emrcontainers/integ.job-submission-workflow.ts

```diff
@@ -0,0 +1,79 @@
+import * as eks from '@aws-cdk/aws-eks';
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import {
+  Classification, VirtualClusterInput, EksClusterInput, EmrContainersDeleteVirtualCluster,
+  EmrContainersEksCreateVirtualCluster, EmrContainersStartJobRun, ReleaseLabel,
+} from '../../lib';
+
+/**
+ * Stack verification steps:
+ * Everything in the link below must be setup before running the state machine.
+ * @see https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up.html
+ *
+ * aws stepfunctions start-execution --state-machine-arn <deployed state machine arn> : should return execution arn
+ * aws stepfunctions describe-execution --execution-arn <exection-arn generated before> : should return status as SUCCEEDED
+ */
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-stepfunctions-tasks-emr-containers-all-services-integ');
+
+const eksCluster = eks.Cluster.fromClusterAttributes(stack, 'EKS Cluster', {
+  clusterName: 'test-eks',
+});
```

Comment:
you cannot use `from` method here since this cluster will not exist in the integration test account

Reply:
We'd have to enable the cluster access, and iam role, etc etc for this.

Reply:
That's fine, detail the instructions at the top of this test. We need these to be actual applications that can deploy and execute; they're not just mocks